web/src/components/task/task-sidebar-flow/index.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FC, useEffect, useMemo, useState } from 'react';
+import { Button, FlexRow, Panel, TabButton } from '@epam/loveship';
+import styles from './styles.module.scss';
```

Comment:
It would be better to move the styles to the bottom of the list of imports to avoid possible overwriting problems from other components. Unfortunately, now not all styles are isolated inside their component and it is important to follow the import order

Reply:
I'm not sure that I understand how any global styles may override our incapsulated(module) styles